person.rb

```diff
@@ -11,6 +11,7 @@ def initialize(age, name, parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
```

Comment:
- [ ] According to the project requirement you need to  . Please make sure to add `method for adding rentals as required for this project . I would like to recommend you to follow this[ link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) to get more about it and make sure that the person itself should be sent as a parameter to create the new rental achieving the 'has-many' association  :+1: